### PR TITLE
Update to uint channel identifier

### PR DESCRIPTION
**Pull request by palango**
Update the channel identifier to `uint`.

### CODE REVIEW

File: pathfinder/api/rest.py [hunk 1]
```diff
@@ -46,10 +46,9 @@ def _validate_token_network_argument(
 
     def _validate_channel_id_argument(self, channel_id: str) -> Optional[Tuple[Dict, int]]:
```
Comment (pcppcp) [inline pathfinder/api/rest.py]: just use ints, no need for silly conversions at this level - they should be done by the caller.

File: pathfinder/tests/test_rest.py [hunk 2]
```diff
@@ -89,12 +89,11 @@ def test_put_balance_sync_check(
     assert response.status_code == 400
     assert response.json()['error'].startswith(
         'The channel identifier from the balance proof '
-        '(0x000000000000000000000000000000000000000000000000000000000000007b) and the request '
-        '(0x000000000000000000000000000000000000000000000000000000000000000c) do not match'
+        '(123) and the request (12) do not match'
```
Comment (pcppcp) [inline pathfinder/tests/test_rest.py]: `'(%d) and the request (%d) do not match' % (ID_12, ID_123)`

File: pathfinder/api/rest.py (hunk 1 above)
Comment (pcppcp) [reply pathfinder/api/rest.py]: (you can use something like

`self.api.add_resource(Resource, API_PATH + "/channels/<str:token_addr>/<int:channel_id>")`

when setting up endpoints)

Comment (pcppcp) [reply pathfinder/api/rest.py]: fair point, I haven't considered this scenario.

**Comment by palango** [comment]
And it seems we need another libs update: https://travis-ci.org/raiden-network/raiden-pathfinding-service/builds/411197090#L868

**Comment by pcppcp** [comment]
you forgot to update contracts commit hash in the `requirements-dev.txt` - it's still using old (str, str) ABI

**Comment by palango** [comment]
None of them is pinned:
```

git+https://github.com/raiden-network/raiden-libs.git
git+https://github.com/raiden-network/raiden-contracts.git
```

**Comment by pcppcp** [comment]
aww, sorry, I was in a wrong directory when checking the file :)

**Pull request merged**